File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/")

    print(driver.title)

    driver.find_element(By.ID, "about-link").click()

    print("About page title is: ", driver.title)

    driver.close()

File: Selenium/Activities/Activity1.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {

	public static void main(String[] args) {
		
		WebDriver driver =  new FirefoxDriver();
		
		driver.get("https://v1.training-support.net/");
		
		System.out.println(driver.getTitle());
		
		driver.findElement(By.id("about-link")).click();
	    
	    System.out.println("About page: " + driver.getTitle());

	    driver.close();
	}

}
